doc/manual/source/interaction/geoserver_ui.rst

```diff
@@ -1,61 +1,55 @@
 .. _geoserver_ui:
 
-GeoServer GUI Configuration
+GeoServer GUI configuration
 ===========================
 
-A GeoServer extension is available to allow GeoServer to interact with a GeoGig repository and use it as data store. It
-enables a GeoGig repository to be exposed as a remote for cloning, pushing and pulling as well as to publish its data via OGC services
-(WMS/WFS/WMTS/etc). Each top level tree (often called "feature tree") in a GeoGig repository corresponds to a GeoServer layer. GeoServer treats a GeoGig
-repository as a store in much the same way as it does a database.
+A GeoServer extension is available to allow GeoServer to interact with a GeoGig repository and use it as a datastore. It enables a GeoGig repository to be exposed as a remote for cloning, pushing, and pulling, as well as to publish its data via OGC services (WMS/WFS/WMTS/etc). Each top-level tree (often called "feature tree") in a GeoGig repository corresponds to a GeoServer layer. GeoServer treats a GeoGig repository configured as a store in much the same way as it does a database.
 
 Building/installing the GeoServer GeoGig extension
 --------------------------------------------------
 
 You can download the latest stable version of the GeoGig GeoServer plugin from the `GeoGig <http://www.geogig.org/>`_ home page.
 
-In order to build it from sources, a GeoGig module is currently included in the 2.8.x branch of GeoServer's community extensions. To build it, clone the
-GeoServer GitHub repository::
-
+In order to build it from sources, a GeoGig module is currently included in the 2.8.x branch of GeoServer's community extensions. To build it, clone the GeoServer GitHub repository.
+::
```

Comment:
for Sphinx RST formatting, there needs to be a blank line after the literal block `::` marker